sites/main-page-title/includes/main-page-title.html

```diff
@@ -1,7 +1,12 @@
-{%- if page.sectionTitle -%}
+{%- if page.sectionTitle or page.byline -%}
 <hgroup id="wb-cont">
+	{%- if page.sectionTitle -%}
 	<p>{{ page.sectionTitle }}</p>
+	{%- endif -%}
 	<h1 property="name">{{ page.title }}</h1>
+	{%- if page.byline -%}
+	<p>{% if page.language == "fr" %}De&nbsp;:{% else %}From:{% endif %} <a href="{{ page.byline.url }}">{{ page.byline.institution }}</a></p>
```

Comment:
The by-line wasn't bolded as expected when I tested

```suggestion
	<p class="gc-byline">{% if page.language == "fr" %}De&nbsp;:{% else %}From:{% endif %} <a href="{{ page.byline.url }}">{{ page.byline.institution }}</a></p>
```